hw/hw3/forms.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Length, EqualTo, Email
+
+
+class RegistrationForm(FlaskForm):
+    name = StringField('Имя', validators=[DataRequired()])
+    surname = StringField('Фамилия', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
+    password = PasswordField('Пароль', validators=[DataRequired(), Length(min=6)])
+    confirm_password = PasswordField('Confirm Пароль', validators=[DataRequired(), EqualTo('password')])
```

Comment:
Если тут добавить поле `SubmitField`
```suggestion
    confirm_password = PasswordField('Confirm Пароль', validators=[DataRequired(), EqualTo('password')])
    submit = SubmitField('Зарегистрироваться')
```